docs/preset-env.md

```diff
@@ -229,40 +229,45 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` imports polyfills and runtime support into your generated code.
 
-When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+Specifically, this option supports the following external packages:
 
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
+- `core-js`: For standard JS built-ins
+- `regenerator-runtime`: For generator and `async`/`await` support (for browsers that lack support)
```

Comment:
```suggestion
- `regenerator-runtime`: For generator and `async`/`await` support (for browsers that lack support for generators)
```

Again, it's important here to note that if a browser supports generators you can use `async` _without_ needing `regenerator-runtime` :sweat_smile: